drydock/patches/caddyfile

```diff
@@ -15,7 +15,7 @@ request_body {
     max_size 4MB
```

Comment:
This PR is starting to get a bit out of control due to changes related to different things. I would propose splitting this PR out into the following PR's:
1. Fixes to the SCORM patches related to the existence of a storage, (either Minio or S3) with the corresponding backports
2. Doc fixes related to the documentation of missing Drydock variables (I'm pretty sure DRYDOCK_INGRESS_LMS_EXTRA_HOSTS is not the only one missing) with backports
3. Integration test PR and backports

NOTE: I think PR's 1 and 2 can be merged, however, I leave it to your criteria